internal/pkg/describe/service.go

```diff
@@ -118,6 +127,30 @@ func (d *ServiceDescriber) Secrets() ([]*ecs.ContainerSecret, error) {
 	return taskDefinition.Secrets(), nil
 }
 
+// NetworkConfiguration returns the network configuration of the service.
+func (d *ServiceDescriber) NetworkConfiguration() (*awsecs.NetworkConfiguration, error) {
+	clusterARN, err := d.clusterDescriber.ClusterARN(d.app, d.env)
+	if err != nil {
+		return nil, fmt.Errorf("get cluster ARN for service %s: %w", d.service, err)
+	}
+
+	service, err := d.ecsClient.Service(clusterARN, d.service)
+	if err != nil {
+		return nil, fmt.Errorf("get service %s running on cluster %s: %w", d.service, clusterARN, err)
+	}
+
+	networkConfig := service.NetworkConfiguration
+	if networkConfig == nil || networkConfig.AwsvpcConfiguration == nil {
+		return nil, fmt.Errorf("unable to retrieve network information for service %s", d.service)
```

Comment:
since we are not wrapping up any returned error here, maybe we want the error message to be more specific?
```suggestion
		return nil, fmt.Errorf("cannot get the awsvpc configuration for service %s", d.service)
```
Sorry I am struggling with a better error massage.